src/cls/rgw/cls_rgw.cc

```diff
@@ -850,31 +858,44 @@ static int read_key_entry(cls_method_context_t hctx, const cls_rgw_obj_key& key,
 int rgw_bucket_prepare_op(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
 {
   CLS_LOG(10, "entered %s", __func__);
+
+  const ConfigProxy& conf = cls_get_config(hctx);
+
+  // bucket index transaction instrumentation
+  const bool bitx_inst =
+    conf->rgw_bucket_index_transaction_instrumentation;
```

Comment:
@adamemerson , @mattbenjamin That code is outdated; this what it now looks like.

Reply:
maybx bit_txn would be clearer, but I like it fine as is